### PR TITLE
feat: add Mail icon

### PR DESCRIPTION
**Pull request by Mayank-CES**
added Mail Icon
image

### CODE REVIEW

File: packages/blade/src/components/Icons/EmailIcon/EmailIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import type { ReactElement } from 'react';
+import type { IconComponent, IconProps } from '..';
+import useIconProps from '../useIconProps';
+import { Path, Svg } from '../_Svg';
+
+const EmailIcon: IconComponent = ({ size, color }: IconProps): ReactElement => {
+  const { height, width, iconColor } = useIconProps({ size, color });
+  return (
+    <Svg width={width} height={height} viewBox="0 0 22 18" fill="none">
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/Icons/EmailIcon/EmailIcon.tsx:9]: Can you pick the correct icon from Figma (root element) and use https://react-svgr.com/playground/?expandProps=none&native=true&svgo=false to convert it into a react component?

If its copied correctly, the viewBox would be `0 0 24 24`


Screenshot 2022-11-22 at 3 05 04 PM

**Review by chaitanyadeorukhkar — CHANGES_REQUESTED** [review]

File: packages/blade/src/components/Icons/EmailIcon/EmailIcon.tsx (hunk 1 above)
Comment (Mayank-CES) [reply packages/blade/src/components/Icons/EmailIcon/EmailIcon.tsx:9]: Its root icon name is "mail". should i create name as MailIcon or EmailIcon is fine

File: .changeset/big-rats-brake.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@razorpay/blade': patch
+---
+
+feat: adding Mail icon
```
Comment (chaitanyadeorukhkar) [inline .changeset/big-rats-brake.md:5]:
```suggestion
feat: add Mail icon
```

**Review by chaitanyadeorukhkar — APPROVED** [review]

File: packages/blade/src/components/Icons/MailIcon/MailIcon.native.test.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import EmailIcon from '.';
```
Comment (divyanshu013) [inline packages/blade/src/components/Icons/MailIcon/MailIcon.native.test.tsx:1]: Let's use the same terminology:

```suggestion
import MailIcon from '.';
```

File: packages/blade/src/components/Icons/MailIcon/MailIcon.native.test.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.native';
+
+describe('<EmailIcon />', () => {
+  it('should render EmailIcon', () => {
+    const renderTree = renderWithTheme(
+      <EmailIcon color="feedback.icon.neutral.lowContrast" size="large" />,
+    ).toJSON();
+    expect(renderTree).toMatchSnapshot();
+  });
+});
```
Comment (divyanshu013) [inline packages/blade/src/components/Icons/MailIcon/MailIcon.native.test.tsx:11]:
```suggestion
describe('<MailIcon />', () => {
  it('should render MailIcon', () => {
    const renderTree = renderWithTheme(
      <MailIcon color="feedback.icon.neutral.lowContrast" size="large" />,
    ).toJSON();
    expect(renderTree).toMatchSnapshot();
  });
});
```

File: packages/blade/src/components/Icons/MailIcon/MailIcon.web.test.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import EmailIcon from '.';
```
Comment (divyanshu013) [inline packages/blade/src/components/Icons/MailIcon/MailIcon.web.test.tsx:1]:
```suggestion
import MailIcon from '.';
```

File: packages/blade/src/components/Icons/MailIcon/MailIcon.web.test.tsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.web';
+
+describe('<EmailIcon />', () => {
+  it('should render EmailIcon', () => {
+    const { container } = renderWithTheme(
+      <EmailIcon color="feedback.icon.neutral.lowContrast" size="large" />,
+    );
+    expect(container).toMatchSnapshot();
+  });
+});
```
Comment (divyanshu013) [inline packages/blade/src/components/Icons/MailIcon/MailIcon.web.test.tsx:11]:
```suggestion
describe('<MailIcon />', () => {
  it('should render MailIcon', () => {
    const { container } = renderWithTheme(
      <MailIcon color="feedback.icon.neutral.lowContrast" size="large" />,
    );
    expect(container).toMatchSnapshot();
  });
});
```

**Review by divyanshu013 — APPROVED** [review]
Looks good to me.

File: packages/blade/src/components/Icons/iconMap.ts [hunk 7]
```diff
@@ -84,6 +84,7 @@ const iconMap: Record<string, IconComponent> = {
   RotateCounterClockWiseIcon: RotateCounterClockWiseIconComponent,
   TrendingUpIcon: TrendingUpIconComponent,
   TrendingDownIcon: TrendingDownIconComponent,
+  MailIcon: MailIconComponent,
```
Comment (akkhil7) [inline packages/blade/src/components/Icons/iconMap.ts:87]: Can we have it at the right alphabetical position? Easier for folks to navigate. Check for other files too.

**Pull request merged**